react-training/src/App.js

```diff
@@ -0,0 +1,16 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
+import Card from "./components/Card";
+
+function App() {
+  return (
+    <div className="container">
+        <div className="jumbotron">
+            <h1 className="display-3">App Name</h1>
+        </div>
+        <Card></Card>
```

Comment:
Please don't use pair tags if there is nothing inside
```suggestion
        <Card />
```